train.py

```diff
@@ -165,7 +195,11 @@ def train():
     def update(engine, batch):
         model.train()
         batch = tuple(input_tensor.to(args.device) for input_tensor in batch)
-        lm_loss, mc_loss = model(*batch)
+        
+        # Migration notes: the models now return not only the loss but also the logits. The exact return values for each model is specified in the docs. 
+        # lm_loss, mc_loss = model(*batch)
+        lm_loss, mc_loss, lm_logits, mc_logits = model(*batch)
+
```

Comment:
GPT2 models return `presents` as well because they use `pasts` for sequential decoding. This should have:
```
if "gpt2" in args.model_checkpoint:
    lm_loss, mc_loss, lm_logits, mc_logits, presents = model(*batch)
else:
    lm_loss, mc_loss, lm_logits, mc_logits = model(*batch)
```